### PR TITLE
Bug 1866051: Effort to left align content filters so they are consistent with pf-c-toolbar

### PR DESCRIPTION
**Pull request by sg00dwin**
https://bugzilla.redhat.com/show_bug.cgi?id=1866051

Screen Shot 2020-08-18 at 10 15 35 AM
Screen Shot 2020-08-18 at 10 27 56 AM

Screen Shot 2020-08-18 at 10 15 22 AM
Screen Shot 2020-08-18 at 10 15 09 AM

### CODE REVIEW

**Comment by gdoyle1** [comment]
@sg00dwin Looks great! Thanks for picking this up!

File: frontend/public/components/RBAC/_rbac.scss [hunk 1]
```diff
@@ -1,7 +1,3 @@
-.rbac-rules-list {
```
Comment (rhamilto) [inline frontend/public/components/RBAC/_rbac.scss:1]: Remove the [className from the jsx](https://github.com/openshift/console/blob/master/frontend/public/components/RBAC/rules.jsx#L14), too?

File: frontend/public/components/cluster-settings/global-config.tsx [hunk 2]
```diff
@@ -134,23 +134,21 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
     });
     return (
       <>
-        {!loading && (
-          <div className="co-m-pane__filter-bar co-m-pane__filter-bar--with-help-text">
```
Comment (rhamilto) [inline frontend/public/components/cluster-settings/global-config.tsx:138]: `co-m-pane__filter-bar--with-help-text` can be removed from the SCSS.

File: frontend/public/components/cluster-settings/global-config.tsx [hunk 3]
```diff
@@ -134,23 +134,21 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
...
-            <div className="co-m-pane__filter-bar-group co-m-pane__filter-bar-group--help-text">
```
Comment (rhamilto) [inline frontend/public/components/cluster-settings/global-config.tsx:139]: `co-m-pane__filter-bar-group--help-text` can be removed from the SCSS.

`co-m-pane__filter-bar-group--badge` can be removed from the SCSS as well (was previously orphaned).

File: frontend/public/components/RBAC/role.jsx [hunk 4]
```diff
@@ -137,17 +137,16 @@ class Details extends React.Component {
         </div>
         <div className="co-m-pane__body">
           <SectionHeading text="Rules" />
-          <div className="co-m-pane__filter-bar co-m-pane__filter-bar--alt">
```
Comment (rhamilto) [inline frontend/public/components/RBAC/role.jsx:140]: `co-m-pane__filter-bar--alt` can be removed from the SCSS.

File: frontend/public/components/cluster-settings/global-config.tsx [hunk 5]
```diff
@@ -134,23 +134,21 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
...
+        <div className="co-m-pane__body">
+          {!loading && (
+            <div>
```
Comment (rhamilto) [inline frontend/public/components/cluster-settings/global-config.tsx:139]:
```suggestion
            <>
```
as you don't need an HTML element here.

**Comment by rhamilto** [comment]
Screen Shot 2020-08-18 at 11 37 42 AM

Text input and button are misaligned < 768px.

File: frontend/public/components/cluster-settings/global-config.tsx (hunk 2 above)
Comment (sg00dwin) [reply frontend/public/components/cluster-settings/global-config.tsx:138]: removed these classes from SCSS

**Pull request merged**